packages/hierarchies-react/src/presentation-hierarchies-react/internal/TreeActions.ts

```diff
@@ -233,7 +232,7 @@ export class TreeActions {
       return;
```

Comment:
maybe return `{ complete: Promise.resolve() }` here and in other places where we do an early return?

Reply:
Empty return should resolve pretty much immediately, I prefer basic return as it is cleaner.